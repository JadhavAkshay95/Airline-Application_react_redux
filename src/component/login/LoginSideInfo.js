import Button from 'react-bootstrap/Button'
import React from 'react'
import { Row, Col } from 'react-bootstrap'
import { faPaperPlane } from '@fortawesome/free-solid-svg-icons'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import PropTypes from 'prop-types'

const LoginSideInfo = ({ showLoginModal }) => {
  return (
    <>
      <Col>
        <Row>
          <Col>
            <div className="flight-intro">
              <h1 className="app-title">
                <FontAwesomeIcon
                  style={{ color: '#00b383', margin: ' 0px 10px' }}
                  icon={faPaperPlane}
                />
                Airline Application
              </h1>
              <h4 style={{ color: '#00b383' }}> Come, fly high with us!</h4>
              <hr></hr>
              <Button onClick={showLoginModal}>Click here to login</Button>
              <hr></hr>
              <h3 className="flight-intro-desc">
                Airlines - Book Domestic and international Airlines in India at
                Lowest airfares at Airline.com. Best deals on Low cost airline
                tickets booking reservation, Airlines operating in India. Get
                cheap fares Airlines Tickets deals online at Make Trip India.
              </h3>
              <hr></hr>
            </div>
          </Col>
        </Row>
        <Row>
          <Col>
            <h5 className="flight-intro-desc-more">
              Airline user friendly navigation helps you find cheap airline
              tickets; in addition to features such as checking check in,
              checkout status, meal pref etc. Come, fly high with us!
            </h5>
          </Col>
        </Row>
        <div></div>
      </Col>
    </>
  )
}

export default LoginSideInfo
LoginSideInfo.propTypes = { showLoginModal: PropTypes.func }
