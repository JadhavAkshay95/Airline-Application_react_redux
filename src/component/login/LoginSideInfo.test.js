import React from 'react'
import { shallow } from 'enzyme'
import LoginSideInfo from './LoginSideInfo'

const renderLoginSideInfo = args => {
  const defaultProps = {
    showLoginModal: () => {}
  }
  const props = { ...defaultProps, ...args }
  return shallow(<LoginSideInfo {...props} />)
}

describe('Render login side info component', () => {
  it('Render login side info', () => {
    const wrapper = renderLoginSideInfo({})
    expect(wrapper).toMatchSnapshot()
  })
})
