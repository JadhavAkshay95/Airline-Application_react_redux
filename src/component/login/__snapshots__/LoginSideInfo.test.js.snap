// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render login side info component Render login side info 1`] = `ShallowWrapper {}`;
